README.md

```diff
@@ -204,13 +204,16 @@ Note: "requirejs" must be included in a global context (usually as a
 
 ## Publishing packages for a JupyterLab release
 
-We publish an npm package, a Python source package, and a Python universal binary wheel.
+We publish an npm package, a Python source package, and a Python universal binary wheel.  
+See the Python docs on [package uploading](https://packaging.python.org/distributing/#uploading-your-project-to-pypi) 
+for twine setup instructions and for why twine is the recommended method.
 
 ```bash
 npm version patch
 git push origin master --tags
 npm publish
 python setup.py sdist
 python setup.py bdist_wheel --universal
+rm -rf dist
```

Comment:
Move that before `python setup.py sdist` I guess, or twine upload will have nothign to upload ?